waku/incentivization/eligibility.nim

```diff
@@ -0,0 +1,6 @@
+import std/options, chronos
+
+import waku/incentivization/[rpc, txid_proof]
+
+proc isEligible*(eligibilityProof: EligibilityProof, ethClient: string): Future[bool] {.async.} =
```

Comment:
Similar comment as before, I think it would help having a description of "elibible" concept
```suggestion
proc isEligible*(eligibilityProof: EligibilityProof, ethClient: string): Future[bool] {.async.} =
   ## We consider a tx to be eligible when...
```